### PR TITLE
proj.db fix backport

### PR DESCRIPTION
**Pull request by nirvn**
~~Hm, turns out we needed this for 2.0. That's a surprise to me. Glad I tested 2.0.15 draft.~~

### CODE REVIEW

**Comment by nirvn** [comment]
Actually, I accidentally updated vcpkg on release-2_0 yesterday (https://github.com/opengisch/QField/commit/28f8315cac5635a6fe4d4fb76fd664dfa4e691b1#diff-afbd2b3775dcda9dd0e87ba2d3e471ed96a0e311d8a3a1d3a744f206a36a6fe2), reverting.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 20deb6459d147eb41823faf7bfca57199df920ff:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2778-0cd32b4878bc381a8511ff8ff9f5b31338075ac5-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2778-0cd32b4878bc381a8511ff8ff9f5b31338075ac5-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2778-0cd32b4878bc381a8511ff8ff9f5b31338075ac5-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2778-0cd32b4878bc381a8511ff8ff9f5b31338075ac5-x86-android.apk)

File: CMakeLists.txt [hunk 1]
```diff
@@ -97,6 +97,10 @@ if (ANDROID)
 
   find_package(Qt5 COMPONENTS AndroidExtras REQUIRED)
 
+  if(NOT EXISTS "${VCPKG_BASE_DIR}/share/proj/data/proj.db")
```
Comment (nirvn) [inline CMakeLists.txt:100]:
```suggestion
  if(NOT EXISTS "${VCPKG_BASE_DIR}/share/proj/proj.db")
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 63593d1e506b2c907603c1af4f1b3776bb32333c:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2778-3dfd4a4004da25d9f5d90cf32db6d0dfa45e2638-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2778-3dfd4a4004da25d9f5d90cf32db6d0dfa45e2638-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2778-3dfd4a4004da25d9f5d90cf32db6d0dfa45e2638-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2778-3dfd4a4004da25d9f5d90cf32db6d0dfa45e2638-x86-android.apk)

**Review by suricactus — APPROVED** [review]

**Pull request merged**